### PR TITLE
33 - Update to Polymer 2.x - part 1

### PR DESCRIPTION
**Pull request by dotpointer**
Update to Polymer 2.x - part 1.

* Upgrade Polymer CLI to 1.8.
* Regenerated yarn.lock and package-lock.json.
* Adding .nvmrc.
* Updated test settings, headless.
* Removing local Firefox test due to geckodriver issue.
* Adding Sauce Labs configuration.
* Updated yarn steps for package-lock.json.

Issue is #33.

### CODE REVIEW

File: test/cosmoz-bottom-bar-view.html [hunk 1]
```diff
@@ -87,7 +87,7 @@
 				bottomBarView = fixture('scrollingBottomBarViewFixture');
 				bottomBar = bottomBarView.$.bottomBar;
 				flushBottomBar(bottomBar, function (){
-					Polymer.Base.async(done, 50);
+					Polymer.Base.async(done, 100);
```
Comment (Wurper) [inline test/cosmoz-bottom-bar-view.html]: Use setTimeout instead here?

File: test/cosmoz-bottom-bar-view.html [hunk 2]
```diff
@@ -68,7 +68,7 @@
 				bottomBarView = fixture('bottomBarViewFixture');
 				bottomBar = bottomBarView.$.bottomBar;
 				flushBottomBar(bottomBar, function (){
-					Polymer.Base.async(done, 50);
+					Polymer.Base.async(done, 100);
```
Comment (Wurper) [inline test/cosmoz-bottom-bar-view.html]: Use setTimeout instead here?

File: test/cosmoz-bottom-bar-view.html (hunk 1 above)
Comment (dotpointer) [reply test/cosmoz-bottom-bar-view.html]: Yes, was going to save it for next pull request, but fixed it now.

File: test/cosmoz-bottom-bar-view.html (hunk 2 above)
Comment (dotpointer) [reply test/cosmoz-bottom-bar-view.html]: Replaced it.

File: test/cosmoz-bottom-bar-view.html (hunk 1 above)
Comment (Wurper) [reply test/cosmoz-bottom-bar-view.html]: Ah, ok!

**Pull request merged**